nrf9120-hal/Cargo.toml

```diff
@@ -0,0 +1,31 @@
+[package]
+name = "nrf9120-hal"
+version = "0.18.0"
+description = "HAL for nRF9120 system-in-package"
+readme = "README.md"
+
+repository = "https://github.com/nrf-rs/nrf-hal"
+authors = [
+    "Jonathan Pallant (42 Technology) <jonathan.pallant@42technology.com>",
```

Comment:
Did you copy this from somewhere else? I think you can put your own details here.

Reply:
Yes.  I started the 9160.  I need to update the README too.  Jumped the gun with this PR...